test/smartseq2_single_sample/pr/ValidateSmartSeq2SingleCell.wdl

```diff
@@ -0,0 +1,43 @@
+
+task ValidateSmartSeq2SingleCell {
+      File counts
+      String expected_counts_hash
+
+      File alignment_summary_metrics
+      String expected_alignment_summary_metrics_hash
+
+  command <<<
+
+    # catch intermittent failures
+    set -eo pipefail
+
+    # calculate hashes; awk is used to extract the hash from the md5sum output that contains both
+    # a hash and the filename that was passed. We parse the first 7 columns because a bug in RSEM
+    # makes later columns non-deterministic.
+    counts_hash=$(cut -f 1-7 "${counts}" | md5sum | awk '{print $1}')
+
+    # the picard metrics are contained in a .tar.gz file; in addition to the above processing,
+    # this unzips the archive, and parses it with awk to remove all the run-specific comment lines (#)
+    picard_metrics_hash=$(cat "${alignment_summary_metrics}" | awk 'NF && $1!~/^#/' | md5sum | awk '{print $1}')
+
+    if [ "$counts_hash" != "${expected_counts_hash}" ]; then
+      >&2 echo "counts_hash ($counts_hash) did not match expected hash (${expected_counts_hash})"
+      fail=true
+    fi
+
+    if [ "$picard_metrics_hash" != "${expected_alignment_summary_metrics_hash}" ]; then
+      >&2 echo "picard_metrics_hash ($picard_metrics_hash) did not match expected hash (${expected_alignment_summary_metrics_hash})"
+      fail=true
+    fi
+
+    if [ $fail == "true" ]; then exit 1; fi
+
+  >>>
+  
+  runtime {
+    docker: "ubuntu:zesty"
+    cpu: 1
+    memory: "3.75 GB"
+    disks: "local-disk 2 HDD"
```

Comment:
as a random piece of information, with Google you cant ask for less than 10 GB (this wont fail it will just be defaulted to 10)